dali/python/nvidia/dali/ops.py

```diff
@@ -404,21 +404,27 @@ def __init__(self, **kwargs):
             self._spec.AddArg("preserve", self._preserve)
             self._preserve = self._preserve or self._schema.IsNoPrune()
 
-            if "output_dtype" in kwargs.keys():
-                if "dtype" in kwargs.keys():
-                    raise TypeError(
-                        ("Operator {} got an unexpected 'output_dtype' argument when " +
-                         "'dtype' is already provided").format(type(self).__name__))
-                # show only this warning
+            # Check for any deprecated arguments that should be replaced or removed
+            arg_names = list(kwargs.keys())
+            for arg_name in arg_names:
+                if not self._schema.IsDeprecatedArg(arg_name):
+                    continue
+                msg = self._schema.DeprecatedArgMsg(arg_name)
+                new_name = self._schema.DeprecatedArgRenamedTo(arg_name)
+                ignore = self._schema.DeprecatedArgIgnore(arg_name)
+                if new_name:
+                    if msg:
+                      msg += " " if msg.endswith('.') else ". "
+                    msg += "Use \"{}\" instead.".format(new_name)
+                    kwargs[new_name] = kwargs[arg_name]
+                    del kwargs[arg_name]
+                elif ignore:
+                    msg += " The argument is no longer used and should be removed."
+                    del kwargs[arg_name]
```

Comment:
If ignore means to not error, than it should be:
```suggestion
                if ignore:
                    del kwargs[arg_name]
```
Also you have some part of the error message defaulted here, which makes it two different places.

Reply:
AFAIK `ignore` means no `longer used`